x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_form.js

```diff
@@ -333,62 +308,49 @@ export const FollowerIndexForm = injectI18n(
       /**
        * Leader index
        */
-      const renderLeaderIndex = () => {
-        const hasError = !!fieldsErrors.leaderIndex;
-        const isInvalid = hasError && areErrorsVisible;
+      const renderLeaderIndex = () => (
+        <FormEntryRow
+          field="leaderIndex"
+          value={followerIndex.leaderIndex}
+          error={fieldsErrors.leaderIndex}
+          schema={follwerIndexFormSchema.leaderIndex}
+          disabled={!isNew}
+          areErrorsVisible={areErrorsVisible}
+          onValueUpdate={this.onFieldsChange}
+          onErrorUpdate={this.onFieldsErrorChange}
+        />
+      );
 
-        return (
-          <EuiDescribedFormGroup
-            title={(
-              <EuiTitle size="s">
-                <h4>
-                  <FormattedMessage
-                    id="xpack.crossClusterReplication.followerIndexForm.sectionLeaderIndexTitle"
-                    defaultMessage="Leader index"
-                  />
-                </h4>
-              </EuiTitle>
-            )}
-            description={(
-              <Fragment>
-                <p>
-                  <FormattedMessage
-                    id="xpack.crossClusterReplication.followerIndexForm.sectionLeaderIndexDescription1"
-                    defaultMessage="The leader index you want to replicate from the remote cluster."
-                  />
-                </p>
-              </Fragment>
-            )}
-            fullWidth
+      /**
+       * Advanced settings
+       */
+      const renderAdvancedSettings = () => (
+        <Fragment>
+          <EuiButtonEmpty
+            iconType={areAdvancedSettingsVisible ? "minusInCircle" : "plusInCircle"}
```

Comment:
minor: what do you think about using `arrowDown` and `arrowUp` instead?

Reply:
I like it, will change